src/OrchardCore.Modules/OrchardCore.Users/Views/Admin/EditPassword.cshtml

```diff
@@ -1,10 +1,17 @@
-@model OrchardCore.Users.ViewModels.ResetPasswordViewModel
+@model OrchardCore.Users.ViewModels.EditPasswordViewModel
 
-<zone Name="Title"><h1>@RenderTitleSegments(T["Edit Password"])</h1></zone>
+<zone Name="Title"><h1>@RenderTitleSegments(T["Change Password"])</h1></zone>
 
 <form asp-controller="Admin" asp-action="EditPassword" method="post" class="form-horizontal no-multisubmit">
     <div asp-validation-summary="All"></div>
     <input asp-for="UsernameOrEmail" type="hidden" />
+        <div class="mb-3">
+            <label asp-for="CurrentPassword" class="col-md-4 form-label">@T["Current password"]</label>
```

Comment:
This could be misleading because the admin may think that they need to provide the user's current password.

Maybe use `Your password` and add a helper text explaining it just to make sure it is not confusing. It would be much better UI if we show a modal to capture the current user's password for confirmation. (e.x, To confirm the change, please provide your password).